platform/ext/target/nordic_nrf/common/core/native_drivers/spu.c

```diff
@@ -72,15 +73,47 @@ void spu_clear_events(void)
     nrf_spu_event_clear(NRF_SPU, NRF_SPU_EVENT_PERIPHACCERR);
 }
 
-void spu_regions_reset_all_secure(void)
+#if defined(PM_MCUBOOT_ADDRESS) || defined(PM_B0_ADDRESS) || defined(PM_S0_ADDRESS) || defined(PM_S1_ADDRESS)
```

Comment:
What are these ifdef's for?

Are we getting an unused function warning?

Reply:
yes